cuqi/samples/_samples.py

```diff
@@ -190,6 +194,24 @@ def geometry(self):
             self._geometry = _DefaultGeometry(grid=np.prod(self.samples.shape[:-1]))
         return self._geometry
 
+    @property
+    def funvals(self):
```

Comment:
We should add a similar method for `.parameters` matching the implementation in CUQIArray.

Reply:
good point! I added this functionality.